zerver/actions/reactions.py

```diff
@@ -50,13 +50,7 @@ def notify_reaction_update(
     # stream the user isn't yet subscribed to; this is the right
     # performance tradeoff to avoid sending every reaction to public
     # stream messages to all users.
```

Comment:
This comment should be removed. And probably a nice version of this logic placed at the top of `event_recipient_ids_for_action_on_messages`